recipe/meta.yaml

```diff
@@ -1,35 +1,37 @@
-{% set version = "2.0.0" %}
+{% set version = "2.1.0" %}
 
 package:
   name: iris
   version: {{ version }}
 
 source:
   url: https://github.com/SciTools/iris/archive/v{{ version }}.tar.gz
-  sha256: 8605ab92c67d622e83fff6d7169fe154d9f8610edd8463b697b574884c158ba2
+  sha256: 9e4cc7ebb41efcd67ece39c39d09fcacadc7ae9d2024fe954bb8149e1b9f7570
 
 build:
-  number: 2
+  number: 0
 
 requirements:
   build:
     - python
     - pip
     - cartopy >=0.14
-    - cf_units
+    - cf_units >=2
```

Comment:
Not sure that is the status of the rename here.

Reply:
@ocefpaf what do you mean?

Reply:
And I was too late :-(